demos/github-demo/src/isograph-components/PullRequestRoute.tsx

```diff
@@ -10,9 +10,9 @@ import {
 import { Container } from '@mui/material';
 
 export const PullRequest = iso(`
-  field Query.PullRequest($repositoryOwner: String!, $repositoryName: String!, $pullRequestNumber: Int!, $last: Int!) @component {
+  field Query.PullRequest($repositoryOwner: String!, $repositoryName: String!, $pullRequestNumber: Int!) @component {
     Header
-    PullRequestDetail
+    PullRequestDetail(repositoryOwner: $repositoryOwner, repositoryName: $repositoryName, pullRequestNumber: $pullRequestNumber)
```

Comment:
😅  I thought we already validated for undefined variables! whoops #134

Reply:
For now I'll leave it to someone else as a first good issue